terms.html

```diff
@@ -11,6 +11,10 @@
   <dd>
 A set of one or more <a>claims</a> made by an <a>issuer</a>.
 The <a>claims</a> in a credential can be about different <a>subjects</a>.
+<p>
+  Our definition of credential differs from, 
```

Comment:
strange place for a comma...
```suggestion
  Our definition of credential differs from
```